cl_sii/cte/f29/parse_datos_obj.py

```diff
@@ -135,6 +135,25 @@ def cte_f29_datos_obj_campo_default_deserializer(campo_value: object, tipo: str)
     return deserialized_value
 
 
+def cte_f29_datos_obj_campo_best_effort_deserializer(campo_value: object, tipo: str) -> object:
+    """
+    Convert raw values from the ``datos`` object to the corresponding Python
+    data type. Values that cannot be converted will be returned without modification.
+
+    :param campo_value: Raw value from 'campos'.
+    :param tipo: Data type code from 'tipos'.
+    """
+    try:
+        deserialized_value = cte_f29_datos_obj_campo_default_deserializer(
+            campo_value=campo_value,
+            tipo=tipo,
+        )
+    except Exception:
```

Comment:
Isn't the exception catch too broad?

Reply:
That's true. At first I was I was only going to catch `ValueError` and `decimal.InvalidOperation`.

Reply:
What changed your mind?

Reply:
The validator is called "best-effort" and I thought that catching only those two exceptions is not the best that can be done.

Reply:
OK